### PR TITLE
(SIMP-5293) Added compliance enforcement check

### PR DESCRIPTION
**Pull request by trevor-vaughan**
* Added compliance enforcement check
* Added compliance report generation check
* Fixed an error in the fixtures file that would cause acceptance test
  hangs

SIMP-5261 #close
SIMP-5262 #close
SIMP-3178 #close
SIMP-3462 #close

### CODE REVIEW

File: .fixtures.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
 fixtures:
   symlinks:
```
Comment (lnemsick-simp) [inline .fixtures.yml]: I don't think you need a .fixtures.yml file at all.  I have other acceptance tests (e.g. simp-utils) for which there is no .fixtures.yml.

**Comment by lnemsick-simp** [comment]
Successfully ran default suite with el7 and el6 node sets, with/without FIPS mode.

**Comment by lnemsick-simp** [comment]
vagrant user cannot login after the compliance test is executed.

File: spec/acceptance/suites/default/10_compliance_enforcement_spec.rb [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  else
+    context 'master setup' do
+
+      let(:compliance_report_hook) { <<-EOS
+          $compliance_profile = 'disa_stig'
+          include 'compliance_markup'
+        EOS
+      }
+
+      let(:manifest) {
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/default/10_compliance_enforcement_spec.rb]: This is not used, so it should be removed.

**Comment by lnemsick-simp** [comment]
The puppet runs after compliance is enforced have errors related to PKI and require a reboot which is not done by the test.  So, it is unclear to me if the manifests are being fully executed.

**Comment by trevor-vaughan** [comment]
@lnemsick-simp What this is testing happens prior to manifest enforcement. A new ticket should be entered for getting a clean run but the test fully examines what we're trying to evaluate at this time.

**Pull request merged**